editoast/src/railjson/mod.rs

```diff
@@ -214,3 +240,12 @@ pub struct Slope {
     pub begin: f64,
     pub end: f64,
 }
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct LoadingGaugeLimit {
```

Comment:
This is never used. You should add a list of loading gauge limit in `TrackSection` struct.